### ISSUE TITLE
Installer package is missing the Newtonsoft.Json assembly

### ISSUE THREAD

**Issue by doug24** [dnGrep/dnGrep#269]
The recent change to check for new versions added a dependency on the Newtonsoft.Json assembly to parse the github version information.  This assembly needs to be added to the installer package.

**Comment by JVimes** [comment]
I think it needs to be added to AppFragment.wxs. The markup in there looks like it was originally generated from a heat.exe command, documented in heat.txt. We should probably run it again and see what changes.

If that doesn't work for some reason, we could follow in the footsteps of dngrep.exe.config, adding a `<component>`, filling in a new id (make it more readable) and guid, and referencing it in the `<Fragment>`:

```xml
<Component Id="cmp3C0B9E685307149FAB49326CFDE9664B" Guid="{8670B4D7-22A8-4204-BD7C-74B331677E44}">
  <File Id="fil9A87CBB83F920A2DCE4E2100C929412F" KeyPath="yes" So urce="$(var.dnGREP.WPF.TargetDir)\dnGREP.exe.config" />
</Component>
<!-- . . . -->
<Fragment>
  <ComponentGroup Id="dnGREPAppFiles">
    <!-- Add a matching reference here -->
    <ComponentRef Id="cmp4F755EB8AEA654DE0F6FB783BD274E79" />
    <ComponentRef Id="cmpD9FA0B07237DE16A4585D46BED20E933" />
    <ComponentRef Id="cmp5D64364BE848D8D880558DFAFA0DE81C" />
    <!-- . . . -->
  </ComponentGroup>
</Fragment>
```

I'm happy to try, but I'm going to be traveling or otherwise busy for the next 10 days or more. Not sure when I'll get to it.

**Comment by doug24** [comment]
Included in build 2.9.53 and later.
